src/Microsoft.DotNet.CodeFormatting.Tests/app.config

```diff
@@ -1,15 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
This file is unnecessary: App.config files don't do anything for library projects by default.